connectors/douban.fm.js

```diff
@@ -1,22 +1,14 @@
 'use strict';
-
-Connector.playerSelector = '.player-wrapper';
-
-Connector.trackArtSelector = '.playing-cover img';
-
-Connector.getTrack = () => {
-	return $('.player-wrapper .middle>div:nth-child(2) a').first().text();
-};
-
-Connector.getArtist = () => {
-	return $('.player-wrapper .link.artist-name').first().text();
-};
-
-Connector.getDuration = () => {
-	let text = $('.time').text();
-	return Util.stringToSeconds(text.substring(1));
-};
-
-Connector.isPlaying = () => {
-	return $('.player-wrapper .buttons .icon-pause').length > 0;
-};
+const update = (attrs) => {
```

Comment:
Is it a certain purpose to add this function?

Reply:
Just for brevity. If it violates something, I will change it.

Reply:
I would keep all connectors written in the same style (the one rest of all are written in).